### ISSUE TITLE
Play infotainment music only in manual mode

### ISSUE THREAD

**Issue by podhrmic** [GaloisInc/BESSPIN-Tool-Suite#1262]
`self._volume` in `infotainment.py` should be updated only if the simulator is *not* in self-drive mode.

@EthanJamesLew not sure what is the best way to propagate this information to the infotainment?

**Comment by dmzimmerman** [comment]
I think we also need to explicitly set it to 0 when entering self-drive mode, right?

**Comment by podhrmic** [comment]
> I think we also need to explicitly set it to 0 when entering self-drive mode, right?

Good point. So we need to add a handler that is executed when we enter self-drive and set the volume to 0.

**Comment by EthanJamesLew** [comment]
> I think we also need to explicitly set it to 0 when entering self-drive mode, right?

Yes, the state machine director has all the necessary context and flow to handle this correctly.
